doc/GettingStarted.tex

```diff
@@ -181,9 +167,9 @@ \subsubsection{Options}
 \item Godley table show values. When ticked, the values of flow
 variables are displayed in the Godley table whilst a simulation is
 running. This will tend to slow down the simulation somewhat.
-\item Godley table output style --- whether $+/-$ or DR/CR (debit/credit)
+\item Godley table output style - whether +/- or DR/CR (debit/credit)
```

Comment:
Better revert these changes. --- means an em-dash, and ordinary hyphen looks wrong in the printed version.
Also wrapping the +/- with $ means it gets typeset in maths mode, and the '-' looks like a minus sign, not a hyphen.